docs/interface.md

```diff
@@ -28,14 +28,32 @@ Returns a dictionary of the session stats.
 **Example**
 
 ```shell
-$ curl -i -X GET http://localhost:8080/session/stats
+$ http GET http://localhost:8080/session/stats
```

Comment:
Could you maybe add something near the top of the document that informs the reader that we're using httpie for examples and a link to the projects website. I'm not certain httpie is as ubiquitous as curl so we should point out where to find it.

Reply:
Yeah, good idea, will do.